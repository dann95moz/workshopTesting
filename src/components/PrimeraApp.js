import React from 'react'

const PrimeraApp = ({ saludo, subtitulo, otro }) => {
    return (
        <div>
            <h1>{saludo}</h1>
            <p>{subtitulo}</p>
            <p>{otro}</p>
        </div>
    )
}

export default PrimeraApp