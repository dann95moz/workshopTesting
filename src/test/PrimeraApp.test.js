import { shallow } from "enzyme"
import PrimeraApp from "../components/PrimeraApp"

describe("Prueba a PrimeraApp/", () => {
    test("validar render de primeraAapp/", () => {

        const saludo = "hola mundo"
        const subtitulo = "esto es una prueba"
        const wrapper = shallow(<PrimeraApp saludo={saludo} subtitulo={subtitulo}
            otro={"hola"}//u for update the snapshop
        />)
        expect(wrapper).toMatchSnapshot()
    })
})