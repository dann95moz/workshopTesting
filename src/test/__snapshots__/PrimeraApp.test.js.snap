// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba a PrimeraApp/ validar render de primeraAapp/ 1`] = `
<div>
  <h1>
    hola mundo
  </h1>
  <p>
    esto es una prueba
  </p>
  <p>
    hola
  </p>
</div>
`;
